keras/utils/data_utils.py

```diff
@@ -473,12 +478,14 @@ class OrderedEnqueuer(SequenceEnqueuer):
         use_multiprocessing: use multiprocessing if True, otherwise threading
         shuffle: whether to shuffle the data at the beginning of each epoch
     """
+    _SEQUENCE_COUNTER = 0
 
     def __init__(self, sequence,
                  use_multiprocessing=False,
                  shuffle=False):
         self.sequence = sequence
-        self.uuid = int(np.random.rand(1)[0] * 255)
+        self.uid = OrderedEnqueuer._SEQUENCE_COUNTER
```

Comment:
Is there a motivation for using a class attribute? Why not a global variable like the others (it behaves like a global variable anyway?)